content/content-de.md

```diff
@@ -1,172 +1,173 @@
+Wir können uns die Zeichnung als 3D Form vorstellen, weil sie sich mit der Zeit bewegt. Wenn du dir vorstellst, dass die Hand von einer Person gezeichnet wird, stellen die drei Dimensionen die Position der Stiftspitze im zeitlichen Moment dar. Die X- und Y-Dimension geben die Position an, die zeitliche Dimension bestimmt die Zeit in diesem Moment.
 
 <canvas id="peace-3d" class="sketch" width=500 height=500></canvas>
 
-Jetzt, wo wir ein 3D Muster haben, können wir nicht die reguläre 2D Sinuswelle benutzen, um sie darzustellen. Gleich wieviele 2D-Wellen wir aufaddieren, wir erhalten niemals was in 3D. Also brauchen wir etwas anderes.
+Jetzt, da wir ein 3D Muster haben, können wir nicht die regulären 2D Sinuswellen benutzen, um es darzustellen. Egal wie viele der 2D Sinuswellen wir addieren, kommen wir nie auf etwas in 3D. Also brauchen wir etwas anderes.
 
-Was wir benutzen können sind die 3d Spiral-Sinuswellen von eben. Wenn wir davon genug aufaddieren dann erhalten wir unser 3D Muster.
+Was wir benutzen können, sind die 3D Spiral-Sinuswellen von eben. Wenn wir davon sehr viele addieren, dann erhalten wir etwas, was aussieht wie unser 3D Muster.
 
-Erinnern Sie sich, diese Wellen sehen aus wie Kreise, wenn wir von vorn draufgucken. Der Name für Muster eines Kreises der sich um einen anderen Kreis bewegt, nennt man Epizykel.
+Merke, diese Wellen sehen von vorn wie Kreise aus. Das Muster eines Kreises, welcher sich um einen anderen Kreis bewegt nennt man Epizyklus.
 
 <canvas id="peace-build-up" class="sketch" width=500 height=500></canvas>
 <input id="peace-build-up-slider" type="range" min="0" max="1" value="1" step="any">
 
-*Benutzen sie den Regler um die Anzahl der Kreise einzustellen.*
+*Benutze den Schieberegler, um die Anzahl der Kreise einzustellen.*
 
-Wie zuvor erhalten wir eine gute Annährerung an unser Muster mit nur ein paar Kreisen. Da es eine ziemlich einfache Form ist, machen Sie die Ecken schärfer um mehr zu sehen.
+Wie zuvor erhalten wir eine gute Annäherung an unser Muster mit nur ein paar Kreisen. Weil dies ein sehr simples Muster ist, werden von den letzten Wellen nur die Ecken etwas geschärft.
 
-Das funktioniert bei allen Zeichnungen, wirklich! Jetzt ist Ihre Chance damit herumzuspielen.
+Das funktioniert wirklich bei allen Zeichnungen! Dies ist deine Chance damit herumzuspielen.
 
 <div class="multi-container">
 <div class="sketch" >
     <canvas id="draw-zone" class="sketch-child" width=500 height=500></canvas>
-    <p id="draw-zone-instruction" class="instruction">Zeichne hier</p>
-    <button id="draw-zone-undo-button" class="button embedded-button">Zurücknehmen</button>
+    <p id="draw-zone-instruction" class="instruction">Zeichne hier!</p>
+    <button id="draw-zone-undo-button" class="button embedded-button">rückgängig</button>
 </div>
```

Comment:
Should this be capitalised? The English is.
(I'll merge this now but if so let me know and I can make the change)

Reply:
Yeah, it should be capitalised. Personally I would prefer it in lowercase but it seems to be best practice to capitalise (e.g Gmail).